### ISSUE TITLE
A better process for people to run for elected positions

### ISSUE THREAD

**Issue by williamkapke** [nodejs/community-committee#31]
The process of nominating yourself is somewhat problematic. Once 1 person has run, all others that run after imply a negative statement. Such as: `I think I could do better than the others` or `I think the others are wrong`. I'm sure, in some cases, those types of statements are intended.

The scenario has similar implications when a person nominates another person.

In politics- these confrontations are on the front lines of the debates- but in an organization that wants to promote an encouraging/welcoming environment, this is not optimal. Conflicting opinions are not a terrible thing for some things (like technical decisions)- but when it relates to an individual's personal performance- it creates an uncomfortable environment/scenario.

I'm confident that many studies have been done on this; SO... does anyone know of an existing process/framework for a non-confrontational way to run for elected positions?

I would like to review some existing concepts before trying to create/discuss our own.

**Comment by bnb** [comment]
Huge +1 on this - builds an adversarial community rather than a unified community. I have ideas/input but would also love to see some existing context if it's available.

**Comment by nebrius** [comment]
I'm not familiar with any existing info on this, but I think this is a great idea and I hope we can find something that works for us!

**Comment by williamkapke** [comment]
There haven't been any examples of existing solutions posted... SO, I'd like to open it up to brainstorming.

Some may have seen the approach I've suggested on https://github.com/nodejs/TSC/issues/259. It's was just an attempt. We'll see how that pans out.

I also think it's important to add that some of us have concluded that an election for a small group (like the TSC) vs a large group (like the Individual Directors) are different scenarios and probably will need different solutions. If making a suggestion- please note which (or both) you feel it can apply to.

**Comment by Trott** [comment]
Spitballing here, but: Maybe "please submit your nominations to this opaque box by such-and-such a date" and the nominations are only revealed after the date passes and there can be no more nominations? Then it's not a case of "I put my hand up because I think you stink" but rather "Oh, we both want to do this, neat, I guess."

**Comment by williamkapke** [comment]
I think a better way to handle nominations.

...actually @nebrius JUST posted the exact thing in https://github.com/nodejs/TSC/issues/259:
> I just had a different idea though. In the future, we could have a private nomination process where everyone submits their candidacy to, say, Mikeal or Tracy, and then all candidates are revealed at the same time after the nomination period ends.

**Comment by nebrius** [comment]
The fact that three of us had pretty much the exact same thought at the same time tells me we may be on to something here.

**Comment by williamkapke** [comment]
If doing private hand raising, we'd need to emphasize that it is **inappropriate** for you to announce it to everyone before the nomination end date. Optionally: go so far as to say it disqualifies you?

**Comment by ashleygwilliams** [comment]
@williamkapke i think that's a good plan but it's a fine line between saying don't announce it or you are disqualified and "if someone finds out and reports you" you are disqualified. while they might sound different in theory, in practice they could end up looking very similar.

inevitably there will be backchannels- and depending on the prominence of people, those backchannels can feel a lot less "back" than they might intend. this is a complicated but not a bad thing. people *should* be talking about running before they choose to, so i want to make sure people feel safe asking questions and talking about running before they decide to. otherwise, we'll be excluding a whole other group of people with this policy. plus, if you are considering running, it is very reasonable to try to get a sense of the community's thoughts about you and your platform before putting yourself on the ballot.

the key thing here is that we want to encourage people to run regardless of who else is running. while we can prevent the sense of primacy that people who put in their names first get,but preventing people from talking about running is not possible and also probably not a good idea, in my opinion.

tl;dr: we should probably define "announce". i'd like it to be pretty strict and small in scope, so as not to restrict the conversations i think are important to have before deciding to run.

**Comment by bnb** [comment]
@williamkapke has this been resolved, or hit a point where we'd benefit from further discussion in a new issue? If so, can we close it? Thanks.
